balrog.wsgi

```diff
@@ -42,3 +54,6 @@ application.config['SENTRY_PROCESSORS'] = ['auslib.util.sentry.SanitizeHeadersPr
 
 if application.config['SENTRY_DSN']:
     sentry.init_app(application)
+
+if newrelic:
+    application = newrelic.agent.wsgi_application()(application)
```

Comment:
Should we get this set-up for the Admin application as well? It runs out of admin.wsgi.